app/src/main/res/layout-sw320dp/activity_registration.xml

```diff
@@ -1,90 +1,133 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.nitsilchar.hp.passwordStorage.activity.RegistrationActivity"
-    android:background="@drawable/background3">
+    android:background="@drawable/gradient"
+    tools:context="com.nitsilchar.hp.passwordStorage.activity.RegistrationActivity">
 
-    <android.support.v7.widget.CardView
+    <TextView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_centerVertical="true"
-        android:padding="50dp"
-        app:cardCornerRadius="5dp">
+        android:layout_marginTop="50sp"
+        android:fontFamily="sans-serif-smallcaps"
+        android:gravity="center"
+        android:text="@string/app_name"
+        android:textColor="@android:color/white"
+        android:textSize="30sp"

```

Comment:
Can we extract this into a dimension resource?

Reply:
Done.